install/kubernetes/helm/istio/charts/security/templates/deployment.yaml

```diff
@@ -88,6 +88,12 @@ spec:
             initialDelaySeconds: 60
             periodSeconds: 60
           {{- end }}
+            {{- if .Values.env }}
```

Comment:
can you send a PR to istio/installer

Reply:
Sure, I send out https://github.com/istio/installer/pull/422. Please take a look. Thanks!